contributors.yaml

```diff
@@ -0,0 +1,277 @@
+  orcid: "https://orcid.org/0000-0003-3967-905X"
+  affiliation: "Kamuzu University of Health Sciences"
+  website: "https://www.linkedin.com/in/wingston-ng-ambi-7950707b"
+  github-username: winnga # to check
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Bingling
+  family-names: She
+  orcid: "https://orcid.org/0000-0001-5976-8987"
+  affiliation: "Imperial College London"
+  website: "https://www.imperial.ac.uk/people/b.she"
+  github-username: BinglingICL
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Matthew
+  family-names: Gillman
+  github-username: mattagape
+  contributions:
+    - Software development
+- given-names: Mosè
+  family-names: Giordano
+  orcid: "https://orcid.org/0000-0002-7218-2873"
+  affiliation: "University College London"
+  website: "https://giordano.github.io/"
+  github-username: giordano
+  contributions:
+    - Software development
+- given-names: Matthew
+  family-names: Graham
+  orcid: "https://orcid.org/0000-0001-9104-7960"
+  affiliation: "University College London"
+  website: "https://matt-graham.github.io"
+  github-username: matt-graham
+  contributions:
+    - Software development
+- given-names: Emmanuel
+  family-names: Mnjowe
+  affiliation: "Kamuzu University of Health Sciences"
+  website: "https://www.linkedin.com/in/emmanuel-mnjowe-067481162"
+  github-username: mnjowe
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Stefan
+  family-names: Piatek
+  orcid: "https://orcid.org/0000-0002-3028-5292"
+  affiliation: "University College London"
+  website: "https://profiles.ucl.ac.uk/67238"
+  github-username: stefpiatek
+  contributions:
+    - Software development
+- given-names: Jonathan
+  family-names: Cooper
+  orcid: "https://orcid.org/0000-0001-6009-3542"
+  affiliation: "University College London"
+  website: "https://profiles.ucl.ac.uk/59377"
+  github-username: jonc125
+  contributions:
+    - Software development
+- given-names: Dimitra
+  family-names: Salmanidou
+  orcid: "https://orcid.org/0000-0002-2147-0787"  # to check
+  affiliation: "University College London"
+  website: "https://www.researchgate.net/profile/Dimitra-Salmanidou"
+  github-username: dimitrasal
+  contributions:
+    - Software development
+- given-names: William
+  family-names: Graham
+  orcid: "https://orcid.org/0000-0003-0058-263X"
+  affiliation: "University College London"
+  website: "https://willgraham01.github.io/"
+  github-username: willGraham01
+  contributions:
+    - Software development
+- given-names: Margherita
+  family-names: Molaro
+  orcid: "https://orcid.org/0000-0002-1841-4274"  # to check
+  affiliation: "Imperial College London"
+  github-username: marghe-molaro
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Eva
+  family-names: Janoušková
+  orcid: "https://orcid.org/0000-0002-4104-0119"  # to check
```

Comment:
@EvaJanouskova can I check if this a correct ORCID for you?

Reply:
Yes, it is correct.